src/main/java/com/redhat/rhjmc/containerjfr/net/SslConfiguration.java

```diff
@@ -145,74 +118,117 @@ Path discoverKeyStorePathInDefaultLocations() {
 
     Pair<Path, Path> discoverKeyCertPathPairInDefaultLocations() {
         Path home = fs.pathOf(System.getProperty("user.home", "/"));
-        Pair<Path, Path> ret = new Pair<>(null, null);
+        Path key = null;
+        Path cert = null;
 
         // discover keyPath
         if (fs.exists(home.resolve("container-jfr-key.pem"))) {
-            ret.left = home.resolve("container-jfr-key.pem");
-        } else {
-            logger.warn("unable to locate container-jfr-key.pem");
+            key = home.resolve("container-jfr-key.pem");
         }
 
         // discover certPath
         if (fs.exists(home.resolve("container-jfr-cert.pem"))) {
-            ret.right = home.resolve("container-jfr-cert.pem");
-        } else {
-            logger.warn("unable to locate container-jfr-cert.pem");
+            cert = home.resolve("container-jfr-cert.pem");
         }
 
-        if (ret.left == null || ret.right == null) {
+        if (key == null || cert == null) {
             return null;
         }
 
-        return ret;
+        return Pair.of(key, cert);
+    }
+
+    HttpServerOptions applyToHttpServerOptions(HttpServerOptions options) {
+        return strategy.applyKeyStoreOptions(options);
+    }
+
+    boolean enabled() {
+        return strategy.enabled();
+    }
+
+    SslConfigurationStrategy keyStoreStrategy(Path path, String password) throws SslConfigurationException {
```

Comment:
Do you have some specific future plans for this method and the others like it? They look like they can just be inlined as it is now.

Reply:
No. As far as I know, these are all what vertx supports. I'll inline these two calls.